R/overloaded.R

```diff
@@ -13,5 +13,10 @@
 #'   x,y,size,LINPACK,V,na.rm,dims,MARGIN,STATS,FUN,check.margin,\dots,r,k,upper.tri,transpose,l,X,INDEX,symmetric,only.values,EISPACK,x1,x2,compact,along,rev.along,new.names,force.array,make.names,use.anon.names,use.first.dimnames,hier.names,use.dnns,nrow,ncol
 #'    arguments as in original documentation
 #'
+#' @details
+#'   Note that, as in R, the LINPACK argument is defunct and silently ignored.
+#'   The argument is only included for compatibility with the base functions
+#'   that call it.
+#'
```

Comment:
@goldingn I wasn't sure where to put this - does the `overloaded.R` file make sense, or should this go in the `functions.R` file, where `chol2inv` is exported? I just wasn't sure which function to provide the details to.

Reply:
functions.R I think, since that's where the helpfile info for that function is

Reply:
Hmmm, when I search `?chol2inv` it gives an option to look for overloaded, so I think it has to go there?

```r
# nolint start
#' @rdname overloaded
#' @export
chol2inv <- function(x, size = NCOL(x), LINPACK = FALSE) {
  UseMethod("chol2inv", x)
}

```